src/openrct2/paint/Paint.h

```diff
@@ -325,6 +362,14 @@ PaintStruct* PaintAddImageAsChildRotated(
 PaintStruct* PaintAddImageAsParentRotated(
     PaintSession& session, const uint8_t direction, const ImageId imageId, const CoordsXYZ& offset,
     const BoundBoxXYZ& boundBox);
+inline PaintStruct* PaintAddImageAsParentRotated(
```

Comment:
Are we sure this would always be called with TrackColours[0]? maybe it would be better to pass the SCHEME as another var?